File: ChickInRun/Assets/EndingNeed/endinglava.js
#pragma strict

public var openMat : Material;
public var originMat : Material;
private var level : int;
public var order : int;
public var orderA : int;
public var orderB : int;
//gate의 변수를 받아오기 위한 오브젝트

function Start () {
	level = 0;
}

function Update () {
	level++;
	if(level ==orderA){
		GetComponent.<Renderer>().material = openMat;
	}
	else if(level ==orderB){
		GetComponent.<Renderer>().material = originMat;
		level=order;
	}
}

File: ChickInRun/Assets/Script/endinglava.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class endinglava : MonoBehaviour
{
    public Material openMat;
    public Material originMat;
    private int level = 0;
    public int order;
    public int orderA;
    public int orderB;

    // Start is called before the first frame update
    void Start()
    {
        
    }

    // Update is called once per frame
    void Update()
    {
        level++;
        if(level == orderA)
        {
            GetComponent<Renderer>().material = openMat;
        }
        else if(level == orderB)
        {
            GetComponent<Renderer>().material = originMat;
            level = order;
        }
    }
}
